package/gargoyle/files/www/basic.sh

```diff
@@ -944,8 +944,12 @@ var isb43 = wirelessDriver == "mac80211" && (!GwifiN) ? true : false ;
 					<span class="col-xs-7">
 						<select class="form-control" id="wifi_encryption2" onchange="setWifiVisibility(); proofreadPass('wifi_pass2', this)">
 							<option value="none"><%~ None %></option>
+							<option value="sae-mixed">WPA3/WPA2 SAE/PSK</option>
```

Comment:
Is this still right? My feedback was that mixed modes don't make sense for connecting to another network.

Reply:
Yes to 364.
Correct me if i'm mistaken, wifi_encryption2 is used only when we are joining an upstream wifi as a client right? So mixed would not make sense here either? Maybe i have this wrong.

Reply:
@lantis1008 yes, agree - gonna update PR again